### ISSUE TITLE
EG.5.1.1 with S:L455F via G22927C (85 seqs, China, Australia, USA, Japan, Singapore, Canada, UK, France)

### ISSUE THREAD

**Issue by NkRMnZr** [cov-lineages/pango-designation#2250]
_Taken from **Branch 5** of Multitask sars-cov-2-variants/lineage-proposals#537, another proposal to feature `G22927C` rather than **HK.3**'s `G22927T`, after #2194_ 

**Defining Mutations:** `EG.5.1.1` > `G22927C(S:L455F)`
**Query:** `C6541T, G22927C, -C745T, -C3653T, -C6317T, -C6327T, -A6659G, -T7111C, -C8074T, -G8768A, -C10702T, -C11779T, -C13968A, -T17811C, -A21137G, -T21748C, -C22482T, -C24023T, -C27741T, -G29734T`
**Earliest seq:** 2023-07-08 (`EPI_ISL_18012368`, Sichuan, China)
**Latest seq:** 2023-08-27 (`EPI_ISL_18213371`, Qinghai, China)
**Sampled Countries:** China (9: Shanghai/5; Liaoning/1; Qinghai/1; Sichuan/1; Taiwan/1), Australia (1: Queensland), USA (1, New York)

<details><summary>Genomes: </summary>

`EPI_ISL_18012368, EPI_ISL_18074450, EPI_ISL_18105834, EPI_ISL_18108480, EPI_ISL_18122582, EPI_ISL_18138248, EPI_ISL_18146109, EPI_ISL_18160385, EPI_ISL_18164789, EPI_ISL_18208589, EPI_ISL_18213371`
</details>

**UShER:**
https://nextstrain.org/fetch/genome.ucsc.edu/trash/ct/subtreeAuspice1_genome_790d_46dd70.json?f_userOrOld=uploaded%20sample&label=id:node_6947948
![details](https://github.com/cov-lineages/pango-designation/assets/125747944/d910bb4d-f3d5-4bef-96ee-544d1860e2ae)

**Trivia:**
- Different nuc change of `S:L455F` to HK.3, probably artifact, but it could also be real. `G>T` has higher frequency than `G>C`.
![flip](https://github.com/cov-lineages/pango-designation/assets/125747944/33f9c256-2a36-44eb-8a67-00ea54ed0571)

**Comment by corneliusroemer** [comment]
Don't think we should create more issues with potentially artefactual mutations.

Please open one when it's clear that it _isn't_ an artefact. This one here looks similar to the other one with mostly Shanghai sequences, very suspicious.

**Comment by FedeGueli** [comment]
restart tracking in the other issue pls @NkRMnZr we will repropose once 50

**Comment by FedeGueli** [comment]
NOW 62 SAMPLES  , 11 COUNTRIES , DOUBLING IN 8 OR 9 DAYS AS HK.3
PING @corneliusroemer PLEASE REOPEN IT.

**Comment by FedeGueli** [comment]
@NkRMnZr UPDATE THE COUNT PLEASE

**Comment by FedeGueli** [comment]
Sorry for the uppercase, not wanted.

**Comment by FedeGueli** [comment]
> > NOW 62 SAMPLES , 11 COUNTRIES , DOUBLING IN 8 OR 9 DAYS AS HK.3 PING @corneliusroemer PLEASE REOPEN IT.
> 
> I forgot to update the filters to the query, should be less than that. It is still tracked and updated on original issue from the other repo.

Thank you deleted the posts indeed

**Comment by FedeGueli** [comment]
This is becoming interesting @corneliusroemer i suggest to designate the biggest among this EG.5.1+FLip lineages

**Comment by FedeGueli** [comment]
@corneliusroemer please reopen and designate this, important SPIKE DIVERSITY:
![Schermata 2023-10-02 alle 10 24 52](https://github.com/cov-lineages/pango-designation/assets/87669813/d14e2812-e51a-4f6a-a488-3b77c49b0aa7)
As you can see above a subbranch of this one accumulated: S:F797L, S:D1153H first and then splitted in two branches one with **S:Q14R** convergently with JG.3 (the fastest lineage) counting 3 in China  and the other with a new mutation **S:Q474K** counting 4 seqs in Uk

![Schermata 2023-10-02 alle 10 28 39](https://github.com/cov-lineages/pango-designation/assets/87669813/07da4400-5a38-4c84-966f-3a42f27acc15)

https://nextstrain.org/fetch/genome-test.gi.ucsc.edu/trash/ct/subtreeAuspice1_genome_test_1e22a_a7d600.json?c=userOrOld&label=id:node_3664367

**Comment by FedeGueli** [comment]
This is becoming big . @corneliusroemer i suggest strongly to designate the two big FLip lineages with G22927C

**Comment by FedeGueli** [comment]
BRANCH 7 DIED

**Comment by FedeGueli** [comment]
225 now it should be designated.

**Comment by FedeGueli** [comment]
@corneliusroemer here the growth advantage query : https://cov-spectrum.org/explore/World/AllSamples/Past3M/variants?nextcladePangoLineage=HK.3*&nucMutations1=C6541T%2CG22927C%2C3653C%2CC6317C%2CG6320G%2CA6659A%2CT7111T%2CC8074C%2CC6327C%2CC745C%2CG8768G%2CC10702C%2CC11779C%2CC13968C%2CT17811T%2CA21137A%2CT21748T%2CC22482C%2CC24023C%2CC27741C&analysisMode=CompareToBaseline& it seems real.

**Comment by FedeGueli** [comment]
Look at direct comparison in Singapore: https://cov-spectrum.org/explore/Singapore/AllSamples/Past3M/variants?nextcladePangoLineage=HK.3*&nucMutations1=C6541T%2CG22927C%2C3653C%2CC6317C%2CG6320G%2CA6659A%2CT7111T%2CC8074C%2CC6327C%2CC745C%2CG8768G%2CC10702C%2CC11779C%2CC13968C%2CT17811T%2CA21137A%2CT21748T%2CC22482C%2CC24023C%2CC27741C&analysisMode=CompareToBaseline&

**Comment by FedeGueli** [comment]
@NkRMnZr you should add -C20132T cause there is an unrelated spanish sequence with that nuc.

**Comment by corneliusroemer** [comment]
Yes, I am convinced now the 22927X is real too, will designate, thanks!
